src/test/java/PostgresDriverWrapperTest.java

```diff
@@ -0,0 +1,46 @@
+import java.sql.Connection;
+import java.sql.SQLException;
+
+import org.apache.arrow.vector.FieldVector;
+import org.apache.arrow.vector.VectorSchemaRoot;
+import org.junit.jupiter.api.AfterEach;
+import org.junit.jupiter.api.Assertions;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+
+class PostgresDriverWrapperTest {
```

Comment:
Unit tests should not rely on external services

Reply:
done